### DISCUSSION TITLE
Enable Shaded Threshold Filled Region in XY Chart

### DISCUSSION THREAD

**Discussion by mbach138** [grafana/grafana#57445] [Dashboards, Panels & Visualizations]
For the XY Chart (https://grafana.com/grafana/plugins/xychart/) enable filled regions for visualizations like those in the Time Series Chart.
 
![image](https://user-images.githubusercontent.com/79272555/197203625-eedf65de-a7ca-4c42-b181-93efe274d3cc.png)
![image](https://user-images.githubusercontent.com/79272555/197203690-1c1c1c9b-2ae3-4bb8-9d10-29692bcf4407.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
